file/file.go

```diff
@@ -24,8 +24,10 @@ import (
 	"os"
 )
 
-// ReadFile is like ioutil.ReadFile.
-func ReadFile(name string) ([]byte, os.FileInfo, error) {
+var ReadFile = readFile
```

Comment:
Could you please add a comment why this is required, otherwise someone can remove it in the future.

Reply:
Thanks. Does the comment look good to you?